Deployment/src/azure.tf

```diff
@@ -0,0 +1,18 @@
+terraform {
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "=2.45.1"
```

Comment:
Please can we use latest version (2.58.0) of azurerm provider.

Reply:
@vijendr13545 Yes we can use latest version for both. I will do the changes for azurerm for terraform we need to check whether latest terraform is installed in build Agent or not. 
@WorkRD can you please check?

Reply:
@ravi102465 @vijendr13545 

hi, we're currently working on getting the ubuntu agents available to the ukhocustomer Azdo, this will let us run terraform containers so we don't need to worry about breaking changes.

Container Images can be found here: https://hub.docker.com/r/ukhydrographicoffice/terraform

Reply:
@WorkRD I have referenced =0.14.9 which is present on build agent. I will add task in backlog to move it deploy task in container. Can we merge this PR and developer can use it.